utils/automation/FoScanner/SpdxReport.py

```diff
@@ -198,10 +198,10 @@ def write_report(self, file_name: str):
     """
     validation_messages: List[ValidationMessage] = \
       validate_full_spdx_document(self.document)
-    for message in validation_messages:
-      logging.warning(message.validation_message)
-      logging.warning(message.context)
-    assert validation_messages == []
```

Comment:
Since if `AssertionError` is raised, then the `write_file` function won't be called at all, I presumed the other correct documents won't be written as well.
But after looking at the docs of spdx_tools library, I see that the recommended approach is not to write the whole SPDX document if there are validation errors.
I will revert these changes accordingly.